lib/Conversion/TorchToLinalg/DataMovement.cpp

```diff
@@ -2610,21 +2611,43 @@ SmallVector<StringRef> ConvertSparseOperatorOp::legalizedNames = {
 };
 } // namespace
 
-void mlir::torch::torch_to_linalg::populateDataMovementPatternsAndLegality(
-    TypeConverter &typeConverter, RewritePatternSet &patterns,
-    ConversionTarget &target) {
-  // Add some legal ops for torch-torch lowering.
+void mlir::torch::torch_to_linalg::populateDataMovementOpsLegality(
+    ConversionTarget &target) { // Add some legal ops for torch-torch lowering.
   target.addLegalOp<ConstantIntOp>();
+  target.addIllegalOp<AtenReflectionPad1dOp>();
```

Comment:
I see we seem to mark only ops from the `Aten` dialect as `illegal` here and in other places, just curious to know if we see conversion failing in some scenario only for `Aten` dialect or we have seen this issue with other dialects too.